zulipterminal/ui_tools/buttons.py

```diff
@@ -94,8 +94,8 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
 
 class HomeButton(TopButton):
     def __init__(self, controller: Any, width: int, count: int=0) -> None:
-        button_text = ("All messages   ["
-                       + keys_for_command("GO_BACK").pop(0)  # FIXME
+        button_text = ("All messages     ["
+                       + keys_for_command("ALL_MESSAGES").pop(0)  # FIXME
```

Comment:
I checked the git log and we can remove this FIXME, which I think I added to indicate that `GO_BACK` should be replaced with something more specific, just like you're doing :+1: